aclk/README.md

```diff
@@ -1,10 +1,120 @@
+The ACLK uses MQTT over secure websockets to create, persist, and encrypt the connect from end-to-end, and enable the 
+features found in Netdata Cloud."
+date: 2020-03-16
 custom_edit_url: https://github.com/netdata/netdata/edit/master/aclk/README.md
 ---
 -->
 
 # Agent-cloud link (ACLK)
 
-This is the agent cloud link (ACLK) information file
\ No newline at end of file
+The agent-cloud link (ACLK) is the mechanism responsible for connecting a Netdata agent to Netdata Cloud. The ACLK uses
+[MQTT](https://en.wikipedia.org/wiki/MQTT) over secure websockets to create, persist, and encrypt the connect from
+end-to-end, and enable the features found in Netdata Cloud.
+
+The ACLK is a vital component of the entirely redesigned and re-architected Netdata Cloud.
+
+_No data is exchanged with Netdata Cloud until you claim a node._ By claiming a node, you opt-in to sending data from
+your agent to Netdata Cloud via the ACLK.
+
+## Claim nodes with the ACLK
+
+To use the ACLK to add a node to Netdata Cloud, you must **claim** the node.
+
+Claiming nodes is a security feature. Through the process of claiming, you demonstrate in a few ways that you have
+administrative access to that node and the configuration settings for its Netdata agent. By logging into the node, you
+prove you have access, and by running the `netdata-cloud.sh` script, you prove you have write access and administrative
+privileges.
+
+The claiming process ensures the ACLK will never be used by a third party to add your node, and thus view your metrics,
+via their Netdata Cloud account.
+
+When you first sign in to the new Netdata Cloud, it will help onboard you to the new platform by asking you to **claim**
+any nodes you want to add to Netdata Cloud. The claiming process involves running a command, provided to you by the
+onboarding process, on each node to establish the token and use the ACLK to create a connection between the node and
+Netdata Cloud.
+
+```bash
+netdata-cloud.sh --token 5AJRdC9H.XytDT8yWZB5psoy.BzCQz5IRm3uqo.nHNl9HQG3nEiTbPBxu_qYW2xdeFsEqFboD2.sf6zVPuveRtK3Xc1ouuDOS9hiCCLJRx1IBvktsvD16zH57tfCYh3kWxjEQH4 --rooms general,web
+```
+
+The `netdata-cloud.sh` command takes two arguments:
+
+-   `--token`: The unique token to your Netdata Cloud account to establish the ACLK.
+-   `--rooms`: A comma-separated list of the War Rooms this node should belong to.
+
+> It may take up to 60 seconds for your node to connect to Netdata Cloud after executing the `netdata-cloud.sh` command.
+
+When a user of a Space visits an agent claimed by that Space, they automatically grant that agent access to their Cloud
+data via the ACLK.
+
+## Enable and configure the ACLK
+
+The ACLK is enabled by default if its prerequisites installed correctly, and the correct configuration is already set in
+your agent's `netdata.conf` file.
+
+> 🎆 Needed: Any other configuration settings?
```

Comment:
We just started discussing this, the current state of the code expects a `[agent_cloud_link]` section but this does not match the section naming conventions and there is no point separating the options. How would it look if we changed the layout of the config to this (@cosmix please check) :
```
[cloud]
  cloud base url = blahbhlahs
  proxy = socks5h://fhdjkfhjd
  server cert = ...
```
I don't like the repetition of `cloud` in both the section name and the setting but it is exposed via the registry / API and would break compatibility if we change it.